src/vmm/src/resources.rs

```diff
@@ -1579,5 +1611,15 @@ mod tests {
                 VsockConfigError::CreateVsockDevice(VsockError::BufDescTooSmall)
             )
         );
+        assert_eq!(
+            Error::EntropyDevice(EntropyDeviceError::CreateDevice(
+                EntropyError::ReadOnlyDescriptor
+            ))
+            .to_string(),
+            format!(
+                "Entropy device error: {}",
+                EntropyDeviceError::CreateDevice(EntropyError::ReadOnlyDescriptor)
+            )
+        );
```

Comment:
I _think_ that these have been implemented because when we used kcov, the report would show that the `Debug` or `Display` was not covered by tests. So this is a way to increase code coverage. Not sure if this is still the case with grcov

Reply:
I believe this is still the case with `grcov`, this cannot be avoided until https://github.com/rust-lang/rust/issues/84605 is closed and `#[no_coverage]` is stabilized, and even then this would only help if these derived implementations used `#[no_coverage]` which would depend on `std`.

Reply:
How many lines will be counted as uncovered? I thought it was only the actual `#[derive(Debug)]` that'd count as uncovered, not the entire implementation. Because in that case imo it'd be justifiable to remove these noisy tests and accept a slight drop in coverage (or, just pass `#[derive` to grcov's `--excl-line` option). 

Either way, also out of scope for this PR.

Reply:
After discussion with @dpopa in https://github.com/firecracker-microvm/firecracker/pull/3251#discussion_r1072328768 (and offline), we do want to remove these tests whenever we switch a manual Display implementation to a thiserror generated one. So this one should go with 904b612eb6975e19340b64c0ba6f923ce5d04c3d

Reply:
Hmmm, should I drop the whole test?

Reply:
Yeah, since the code it was "testing" isnt there anymore, I'd say so